extensions/window/window_switcher.lua

```diff
@@ -224,11 +224,11 @@ local function exit(self)
   --  if windows[selected]:application():bundleID()~='com.apple.finder' then
   --    windows[selected]:focus()
   --  else
-  timer.doAfter(0.15,function()windows[selected]:focus()end) -- el cap bugs out (desktop "floats" on top) if done directly
+  timer.doAfter(0.01,function()windows[selected]:focus()end) -- el cap bugs out (desktop "floats" on top) if done directly
```

Comment:
FWIW El Capitan is no longer supported by Hammerspoon, so we could see what happens if we remove the `timer.doAfter()` wrapper and just directly call `windows[selected]:focus()`

Reply:
I just tested. Directly calling it seems to work well for me. Should I change it?

Reply:
Apologies for not replying - yeah I think we should change it.

Reply:
Ok, I just committed the change